### ISSUE TITLE
[bug] WebviewWindowBuilder fails on Windows 10

### ISSUE THREAD

**Issue by 1zun4** [tauri-apps/tauri#9252]
### Describe the bug

On Windows 10, a WebView created using WebviewWindowBuilder opens and immediately closes without throwing an error.
It can only be detected by checking for `is_visible`, which results in a `failed to receive message from webview` error.

https://github.com/tauri-apps/tauri/assets/12410754/7c22a6d3-9215-4df7-add0-eeb955468578

### Reproduction
- Clone https://github.com/1zun4/tauri_w10_webview
- Run `yarn` and `yarn tauri dev`
- Click `Open WebView` button

### Expected behavior

Opens a web view, user must follow the download process, application waits for actual download link and then continues.

From my Windows 11 VM (with my production software):
![image](https://github.com/tauri-apps/tauri/assets/12410754/ee5247ab-3dc6-4bf6-9044-df99df278015)

### Full `tauri info` output

```text
yarn tauri info
yarn run v1.22.22
$ tauri info

[✔] Environment
    - OS: Windows 10.0.19045 X64
    ✔ WebView2: 122.0.2365.92
    ✔ MSVC: Visual Studio Community 2022
    ✔ rustc: 1.79.0-nightly (0ad927c0c 2024-03-21)
    ✔ cargo: 1.79.0-nightly (d438c80c4 2024-03-19)
    ✔ rustup: 1.27.0 (bbb9276d2 2024-03-08)
    ✔ Rust toolchain: nightly-x86_64-pc-windows-msvc (default)
    - node: 20.11.1
    - yarn: 1.22.22
    - npm: 10.2.4

[-] Packages
    - tauri [RUST]: 2.0.0-beta.13
    - tauri-build [RUST]: 2.0.0-beta.10
    - wry [RUST]: 0.37.0
    - tao [RUST]: 0.26.1
    - @tauri-apps/api [NPM]: 2.0.0-beta.5
    - @tauri-apps/cli [NPM]: 2.0.0-beta.9

[-] App
    - build-type: bundle
    - CSP: unset
    - frontendDist: ../dist
    - devUrl: http://localhost:5173/
    - framework: Svelte
    - bundler: Vite
Done in 11.04s.
```

### Additional context

I will try to investigate further.

**Comment by 1zun4** [comment]
So follow up. I usually tried to replicate this issue using Windows 11 on VM and actual hardware. But the pattern was always people using Windows 10, so I downloaded the ISO and set-up a fresh VM and on my first try I was finally able to replicate the issue:
https://www.youtube.com/watch?v=x2LVqPnVGqo

I will report back when I find the exact reason.

![image](https://github.com/tauri-apps/tauri/assets/12410754/e77416bd-7848-4005-b9e2-b9c22b842877)

**Comment by 1zun4** [comment]
I have now updated the problem description with a demo to reproduce the problem. Looking forward to your help! 👍

**Comment by 1zun4** [comment]
Demo:
- with SmartScreen disabled
- with Windows Defender disabled

https://github.com/tauri-apps/tauri/assets/12410754/ddbc5c69-efb6-4e82-8525-7fa165ef159b

**Comment by 1zun4** [comment]
Another thing I have now tried was starting the second webview on start-up:

```json
{
    ...
    "app": {
        "withGlobalTauri": true,
        "windows": [
            {
                "title": "LiquidLauncher",
                "width": 1000,
                "minWidth": 1000,
                "height": 670,
                "minHeight": 670,
                "resizable": true,
                "fullscreen": false,
                "transparent": true,
                "decorations": false,
                "center": true,
                "windowEffects": {
                    "effects": [
                        "acrylic"
                    ]
                },
                "shadow": true
            },
            {
                "title": "LiquidBounce Download",
                "label": "downloadView",
                "url": "https://liquidbounce.net/",
                "visible": true,
                "resizable": true,
                "maximized": true,
                "alwaysOnTop": true,
                "center": true,
                "shadow": true
            }
        ]
    },
    ...
}
```

The issue will also occur without any code involved.

**Comment by 1zun4** [comment]
https://github.com/tauri-apps/tauri/blob/dev/examples/multiwindow/ has the same issue.

![image](https://github.com/tauri-apps/tauri/assets/12410754/13a6f8ee-aa80-42bf-ab24-a1d5eccf8f6a)

No secondary window appears.

Tauri v1 in comparison:
![image](https://github.com/tauri-apps/tauri/assets/12410754/2221023e-b86a-449c-89e7-e182ceda0bdb)

**Comment by Zack-Xb** [comment]
Yes, I believe this is being fixed in the new version of `wry`. This is now working well on that branch but the main window now closes when the second window is closed. You can follow the issue here #9199 . I'm going to do some testing to see how this evolved this evening.

**Comment by amrbashir** [comment]
I couldn't reproduce the crash using the reproduction, so I will close this for now since it maybe related to #9199. If the issue still persists after the next release, please let me know

**Comment by Gofive** [comment]
> I couldn't reproduce the crash using the reproduction, so I will close this for now since it maybe related to #9199. If the issue still persists after the next release, please let me know
Does this fixed in tauri v2.0.0-beta.14?
